### DISCUSSION TITLE
Prevent prisma reset on production

### DISCUSSION THREAD

**Discussion by frahman5** [prisma/prisma#3286] [Q&A] [answered]
Is it possible to add a safeguard on our production database that would prevent it from getting reset via `prisma reset`? 

We recently had an incident on our team involving this. Luckily we were able to restore the database, but we're looking for ways to prevent it from happening again. It was a little too easy to accidentally delete the entire production database, so we're hoping for a technical means by which to make it harder to do in the future.

**Comment by janpio** [comment, marked as answer]
`prisma reset` is a Prisma 1 feature, not Prisma 2 (yet). You will want to ask this question in https://github.com/prisma/prisma1/

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
